### DISCUSSION TITLE
Official Helm Chart

### DISCUSSION THREAD

**Discussion by alexanderursu99** [apache/airflow#14039] [Q&A] [answered]
Is this repository the home of the official Helm chart for Airflow?

I notice that [this repository](https://github.com/airflow-helm/charts) also exists, and I happened to find this first on artifacthub.

Does anyone know the difference between these two repos, and which Helm chart should be used?

I noticed that the instructions in this repo say to use `https://charts.helm.sh/stable/`, which I believe is deprecated, whereas the other repo I linked says to use `https://airflow-helm.github.io/charts`.

**Comment by potiuk** [comment, marked as answer]
The chart in apache/airflow is not yet released officially  - many people though are using it in production and the community is planning to start working on the chart after we release 2.0.1 airflow (first bugfix release of 2.0). The other charts are not "official' - in the sense, that they are not maintained by the Airflow community. 

I'd encourage you to help to test and improve the chart in https://github.com/apache/airflow/tree/master/chart  - the more people will test and contribute, the faster we release it 'officially'.

**Reply by alexanderursu99** [reply]
Thanks for the info!

I'll definitely take a look and try it out in the coming days.

Is there any rough timeline for what you mentioned?

**Reply by potiuk** [reply]
Not yet.
